README.md

```diff
@@ -37,3 +37,17 @@ Add `dev.data.gov.uk` after kubernetes.docker.internal on the sameline in the `/
   - See the cluster being created and running
 
   `kubectl get pods`
+
+### Useful tips
+
+#### How to test chart changes
+
+In order to test the changes in EKS follow these steps:
+
+- disable the auto-sync in the `dgu-app-of-apps` application.
```

Comment:
I'd recommend not documenting this here, since it's not something that people should be doing (even in non-prod) once this is in production. It's usually much safer and more convenient to test by [installing the chart with Helm](https://github.com/alphagov/govuk-helm-charts#installing-an-application-chart-without-argo-cd) (under a different name so it doesn't clash with the Argo-managed one).

OTOH it's fine to do this for now while you're writing these charts (i.e. before launch), because if you forget to turn auto-sync back on then it doesn't really affect anyone else.

Reply:
OK, I'll bear this in mind, just thought it would be good to add it here whilst I am testing out the argo flow